src/InstagramBasic/Provider.php

```diff
@@ -75,7 +75,6 @@ protected function mapUserToObject(array $user)
             'email'        => null,
             'avatar'       => null,
             'account_type' => $user['account_type'],
-            'media_count'  => $user['media_count'],
```

Comment:
Should we leave this with a null coalesce?

Reply:
Yeah, fair call.